src/App.js

```diff
@@ -1,14 +1,20 @@
 import "./App.css";
-import { Route, Routes } from "react-router-dom";
+import { Route, Routes, useParams } from "react-router-dom";
 import Home from "./components/Home/Home";
 import Articles from "./components/Articles/Articles";
+import Topics from "./components/Topics/Topics";
+import Header from "./components/Header/Header";
+import Navbar from "./components/Navbar/Navbar";
 
 function App() {
   return (
     <div>
+      <Header />
+      <Navbar />
       <Routes>
         <Route path="/" element={<Home />} />
         <Route path="/articles" element={<Articles />} />
+        <Route path="/articles/:topic" element={<Articles />} />
```

Comment:
If you are using the same the same component and the same kind of route, see if you can remove line 16 and have it still work the same